src/initialization.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const distance = require('ml-distance-euclidean');
+
+/**
+ * Choose K different random points from the original data
+ * @ignore
+ * @param {Array<Array<Number>>} data - Points in the format to cluster [x,y,z,...]
+ * @param {Number} K - Number of clusters
+ * @return {Array<Array<Number>>} - Initial random points
+ */
+function random(data, K) {
+    let len = data.length;
+    let ans = new Array(K);
+
+    for (let i = 0; i < K; ++i) {
+        let rand = Math.floor(Math.random() * len);
```

Comment:
You use this expression 3 times in the code. Could be factored out in a function